### PR TITLE
changed the arduino code to support cleaner current transitions

### PR DESCRIPTION
**Pull request by biocubed**
Made the catheter gui a ros-package to leverage rosdocs and roslint

### CODE REVIEW

**Comment by biocubed** [comment]
@tipakorng , when this PR is approved, I will move the repo to the cwru_mercis group.

File: src/catheter_arduino_ard_ide/catheter_arduino_ard_ide.ino [hunk 1]
```diff
@@ -77,12 +86,16 @@ void setup()
 	camera_counter = 0;
   mriStatOld = false;
 
+  scanDuration = ceil(scanLines*2.88+0.5);
```
Comment (tipakorng) [inline src/catheter_arduino_ard_ide/catheter_arduino_ard_ide.ino]: Could you add a comment explaining this?

File: src/catheter_arduino_gui/CMakeLists.txt [hunk 2]
```diff
@@ -4,24 +4,16 @@ cmake_minimum_required(VERSION 2.8)
 project(catheter_arduino_gui)
 
 
-find_package(catkin)
+find_package(catkin REQUIRED COMPONENTS roslint roscpp)
```
Comment (tipakorng) [inline src/catheter_arduino_gui/CMakeLists.txt]: We don't need roscpp, right?

**Comment by tipakorng** [comment]
Tested the code with the prototype. Worked fine.

**Pull request merged**